### PR TITLE
Urmat: Homework Streams

### PR DESCRIPTION
**Pull request by uri-bbb**

### CODE REVIEW

File: 03-GenericsLambdaStream/src/main/java/homework/urmat/StreamTasks.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        Stream.of("2", 3, "Jack").forEach(object -> {
+            if (object.equals("2"))
+                System.out.println(object);
+        });
+
+        String s = "The quick brown fox and the quick blue hare.";
+        String[] strings = s.split(" ");
+        String modifiedS = Stream.of(strings)
+                .distinct()
+                .filter(str -> !str.equals("the"))
```
Comment (kykaparov) [inline 03-GenericsLambdaStream/src/main/java/homework/urmat/StreamTasks.java:42]: good, but in final result, you should have only one "quick" :)

Comment (uri-bbb) [reply 03-GenericsLambdaStream/src/main/java/homework/urmat/StreamTasks.java:42]: agai, there is one "quick" in the output, getting rid of the extra one with the help of distinct()

Comment (kykaparov) [reply 03-GenericsLambdaStream/src/main/java/homework/urmat/StreamTasks.java:42]: Ahh now I see 👍

**Comment by kykaparov** [comment]
Urmat, this PR is ready to be merged

**Pull request merged**